build.sbt

```diff
@@ -50,10 +50,29 @@ lazy val aggregatedProjects: Seq[ProjectReference] = Seq(
   discovery
 )
 
+/** TODO fix remoteTests in job conf and add remote coverage back.
+  * Does not include
+  * - planned removals in 2.6 https://github.com/akka/akka/milestone/119
+  * - docs?, protobuf, benchJmh
```

Comment:
What's the feeling on docs for coverage inclusion

Reply:
Hmm, yeah, maybe we should rather move things out of the docs package, even when it is primarily for documentation if it is tests, I think we moved in that direction already with the typed modules, better to have tests close to the sources when/if they fail.